src/client/v2/serviceClient.ts

```diff
@@ -66,6 +68,7 @@ export default abstract class ServiceClient {
   }
 
   /**
+   * @ignore
```

Comment:
Why ignore this? It's a useful method

Reply:
I thought developers using the sdk probably don't need to know about it